lib/stdio/sync.js

```diff
@@ -24,7 +25,7 @@ const forbiddenIfSync = ({type, optionName}) => {
 
 const addPropertiesSync = {
 	input: {
-		filePath: ({value}) => ({value: readFileSync(value, 'utf8'), type: 'stringOrBuffer'}),
+		filePath: ({value}) => ({value: bufferToUint8Array(readFileSync(value)), type: 'uint8Array'}),
```

Comment:
Performance optimization. Reading files as buffers instead of serializing to UTF-8 strings is faster.